README-hi.md

```diff
@@ -0,0 +1,82 @@
+# कुबरनेट्स प्रलेखन
```

Comment:
Should this be कुबरनेटिस? Both the words Kubernetes and κυβερνήτης when pronounced sound more like कुबरनेटिस not कुबरनेट्स. I'm not sure though, but that's how I pronounce it.

Reply:
For example, Contributor is written `योगदानकर्ता` but there another way too `योगदानकरता` which is not a good way of writing.